ocp-metadata/ocp-metadata.go

```diff
@@ -270,3 +293,110 @@ func (meta *Metadata) getSDNInfo() (string, error) {
 	}
 	return networkType, err
 }
+
+func (meta *Metadata) getPublish() (string, error) {
+	installConfig, err := meta.getClusterConfig()
```

Comment:
You're calling getClusterConfig many times along the different methods, I wonder if we can call this function only once